src/clients.ts

```diff
@@ -21,6 +21,11 @@ const interceptor = (client) => new Proxy({}, {
   },
 })
 
+export const accountRegistry = (acc: string): Registry => {
```

Comment:
could be only `account` :)

export const accountRegistry = (account: string): Registry => {
     if (!account) account = getAccount();
   return Registry({...options, account, endpoint: endpoint('registry')})
}